extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -562,6 +564,16 @@ export class AmpStoryPage extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Adds icon and pulse animation to bling links
+   */
+  addStylesToBlingLinks_() {
+    scopedQuerySelectorAll(this.element, BLING_LINK_SELECTOR).forEach(el => {
+      AmpStoryBlingLink.addPulseElement(el);
+      AmpStoryBlingLink.addIconElement(el);
```

Comment:
I think we could wrap these two lines in a `AmpStoryBlingLink.build(el)`, where we call the `addPulseElement` & `addIconElement` methods.

In this build method, we could also check whether the bling link has been previously built already by using a custom property in the element to store this information. E.g.

```js
// amp-story-bling-link.js 
build(el) {
  if (el[AMP_BLING_LINK_BUILT]) {
     return;
  } 
  this.addPulseElement(el);
  this.addIconElement(el);
  el[AMP_BLING_LINK_BUILT] = true;
}
```

This way we can save doing those `querySelector` for each link